CHANGELOG.md

```
@@ -15,139 +15,33 @@
+- 
+  [``] by 
+- 
+  [`* https://github.com/piqueterron/OtherMediator/pull/9#issuecomment-3199331614`] by 
+- 
+  [``] by 
+- 
+  [`The following files were modified:`] by 
+- 
+  [``] by 
+- 
+  [`* `src/OtherMediator.Contracts/Unit.cs``] by 
```

markdownlint-cli2 (0.17.2)

Spaces inside code span elements

(MD038, no-space-in-code)